### DISCUSSION TITLE
graphql/envelop/plugins/use-prometheus

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/envelop#1607] [Docs Discussions]
# graphql/envelop/plugins/use-prometheus

GraphQL Envelop Documentation

https://the-guild.dev/graphql/envelop/plugins/use-prometheus

**Comment by martin-cycle** [comment]
Any pre-built grafana dashboard we can use linked to this package ?

**Reply by baudicj** [reply]
very interested !

**Reply by dotansimha** [reply]
> Any pre-built grafana dashboard we can use linked to this package ?

Not at the moment, but please share with us if you are building one so we can collaborate! :)

**Reply by martin-cycle** [reply]
Hey ! I did the dashboard eventually, it is available here 👉 https://grafana.com/grafana/dashboards/?search=graphql+envelop
